.circleci/generate.sh

```diff
@@ -34,6 +34,7 @@ print_help()
   echo "   -a Generate the config.yml, config.yml.FREE and config.yml.PAID expanded configuration"
   echo "      files from the main config_template.yml reusable configuration file."
   echo "      Use this for permanent changes in config.yml that will be committed to the main repo."
+  echo "   -d Deduplicate. Removes all config variations for each test groups leaving only the vanilla one to run. Skinny test run."
```

Comment:
nit: I would remove the sentence "Skinny test run".

Reply:
Instead, we can say this can be used during development for minimum sanity check if we are not ready for full pre-commit CI run.